internal/bus/fake_message_pipe.go

```diff
@@ -90,6 +103,9 @@ func (p *FakeMessagePipe) Run(ctx context.Context) {
 func (p *FakeMessagePipe) RunWithoutInit(ctx context.Context) {
 	var message *Message
 
+	// p.messagesLock.Lock()
```

Comment:
assume this was left in or commented out by mistake

Reply:
updated